server/.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = false
+trim_trailing_whitespace = true
+
+[*.py]
+indent_style = space
+indent_size = 4
+charset = utf-8
+max_line_length = 80
```

Comment:
Can you set something like 80 and above = warning and 120 = hard limit. I am okay with aiming for 80 but sometimes it makes statements less readable imo.

Reply:
There's no setting for warnings or hard limit. It is either one or the other. Which would you prefer? I am a very big fan of 80 since my editor fits exactly two files next to each other that way.

Reply:
Is this just for auto-formatting and warnings or will this also fail CI/CD if not compliant?

If it's just for formatting and warnings, let's do 80.

Reply:
AFAIK there is no CI/CD set up to check code style. Keep in mind that the code style guidelines are used to not include formatting changes in commits which make reviews harder. Therefore deviations from the given code style should be treated as errors.